### PR TITLE
Rba updated demo values

### PR DESCRIPTION
**Pull request by warmkesselj**
### Summary of changes
Updated demo values for RBA that succeed.
### Checklist

- [ ] Added a changelog entry

### Authors
@jwarmkessel 
@KunJeongPark 

-

### CODE REVIEW

**Comment by sarahkoop** [comment]
I think the base of this PR should be the feature branch

**Review by agedd — APPROVED** [review]

File: Demo/Application/Features/PayPalWebCheckoutViewController.swift [hunk 1]
```diff
@@ -135,31 +135,31 @@ class PayPalWebCheckoutViewController: PaymentButtonBaseViewController {
...
             let billingCycle = BTPayPalBillingCycle(
                 interval: .month,
                 intervalCount: 1,
-                numberOfExecutions: 12,
-                sequence: 9,
-                startDate: "2024-04-06T00:00:00Z",
-                isTrial: false,
+                numberOfExecutions: 1,
+                sequence: 1,
+                startDate: "2024-08-01",
```
Comment (sarahkoop) [inline Demo/Application/Features/PayPalWebCheckoutViewController.swift:148]: We should change the doc string associated with this input then - I think we are currently telling merchants to use the previous format

File: Demo/Application/Features/PayPalWebCheckoutViewController.swift [hunk 2]
```diff
@@ -135,31 +135,31 @@ class PayPalWebCheckoutViewController: PaymentButtonBaseViewController {
...
-                totalAmount: "35.99",
+                totalAmount: "32.56",
                 productName: "Vogue Magazine Subscription",
                 productDescription: "Home delivery to Chicago, IL",
                 productQuantity: 1,
-                oneTimeFeeAmount: "5.99",
+                oneTimeFeeAmount: "9.99",
                 shippingAmount: "1.99",
-                productAmount: "9.99",
+                productAmount: "19.99",
```
Comment (sarahkoop) [inline Demo/Application/Features/PayPalWebCheckoutViewController.swift:162]: Are there requirements that these values add up? We should also include that in the doc strings

File: Demo/Application/Features/PayPalWebCheckoutViewController.swift (hunk 1 above)
Comment (jaxdesmarais) [reply Demo/Application/Features/PayPalWebCheckoutViewController.swift:148]: ➕ 1 on updating the doc strings for `startDate`, let also update `billingCycles` to more explicitly call out a max of 3

File: Demo/Application/Features/PayPalWebCheckoutViewController.swift (hunk 2 above)
Comment (KunJeongPark) [reply Demo/Application/Features/PayPalWebCheckoutViewController.swift:162]: I think it was something Justin investigated in Slack messages, nothing officially documented from what I see.

File: Demo/Application/Features/PayPalWebCheckoutViewController.swift (hunk 1 above)
Comment (KunJeongPark) [reply Demo/Application/Features/PayPalWebCheckoutViewController.swift:148]: Addressed on this commit: f89d8f94abde78f94eea294ac47d9c3fedb09770

Comment (KunJeongPark) [reply Demo/Application/Features/PayPalWebCheckoutViewController.swift:148]: @jaxdesmarais Do you mean `billlingCycles` array field in `BTPayPalRecurringBillingDetails` has max of three items? Sorry, I didn't catch context for this.

Comment (KunJeongPark) [reply Demo/Application/Features/PayPalWebCheckoutViewController.swift:148]: Mentioned max number of` billlingCycles` array items in docStrings in this commit: a27721ec785442b32452343d81a0c4e49ccb7d05

File: Sources/BraintreePayPal/RecurringBillingMetadata/BTPayPalRecurringBillingDetails.swift [hunk 3]
```diff
@@ -20,7 +20,7 @@ public struct BTPayPalRecurringBillingDetails {
     
     /// Initialize a `BTPayPalRecurringBillingDetails` object.
     /// - Parameters:
-    ///   - billingCycles: An array of billing cycles for trial billing and regular billing. A plan can have at most two trial cycles and only one regular cycle.
+    ///   - billingCycles: An array of billing cycles for trial billing and regular billing. A plan can have at most two trial cycles and only one regular cycle. Exceeding 3 items in this array results in error.
```
Comment (jaxdesmarais) [inline Sources/BraintreePayPal/RecurringBillingMetadata/BTPayPalRecurringBillingDetails.swift:23]:
```suggestion
    ///   - billingCycles: An array of billing cycles for trial billing and regular billing. A plan can have at most two trial cycles and only one regular cycle. Exceeding 3 items in this array results in an error.
```

Comment (KunJeongPark) [reply Sources/BraintreePayPal/RecurringBillingMetadata/BTPayPalRecurringBillingDetails.swift]: Updated here: a0a3f09a8444a1304b70e3d0a3cbb3f52e249158

**Review by jaxdesmarais — APPROVED** [review]

**Comment by jaxdesmarais** [comment]
@KunJeongPark @jwarmkessel is this ready to be merged since it has 2 approvals or are we waiting on something? Also lets make sure to update the base branch with main post merge.

**Comment by warmkesselj** [comment]
> @KunJeongPark @jwarmkessel is this ready to be merged since it has 2 approvals or are we waiting on something? Also lets make sure to update the base branch with main post merge.

Yeah, was leaving it open for other updates but we can merge.

**Pull request merged**